src/univalue/include/univalue.h

```diff
@@ -99,6 +99,7 @@ class UniValue {
 
     bool push_back(const UniValue& val);
     bool push_backV(const std::vector<UniValue>& vec);
+    UniValue& emplace_back(VType type) { values.emplace_back(type); return values.back(); };
```

Comment:
We need to update the header for this file.

Reply:
Good point, done, thanks!